src/xdsl/mlir_converter.py

```diff
@@ -46,6 +47,29 @@ def convert_type(self, typ: Attribute) -> mlir.ir.Type:
         if isinstance(typ, TupleType):
             return mlir.ir.TupleType.get_tuple(
                 [self.convert_type(t) for t in typ.types.data])
+        if isinstance(typ, LLVMStructType):
+            from xdsl.printer import Printer
+            from io import StringIO
+            type_string = "("
+            for t in typ.types.data:
+                file = StringIO("")
+                p = Printer(stream=file)
+                p.print_attribute(t)
+                file.write(", ")
+                type_string += file.getvalue()[1:]
+            if len(typ.types.data) > 0:
+                type_string = type_string[:-2]
+            type_string += ")"
+
+            if typ.struct_name.data != "":
+                mlir_module = mlir.ir.Module.parse(
```

Comment:
Yeah... I have no clue on how to do this better, so for me that's fine as it is!